src/ncp/ncp_base.cpp

```diff
@@ -208,6 +208,11 @@ const NcpBase::GetPropertyHandlerEntry NcpBase::mGetPropertyHandlerTable[] =
     { SPINEL_PROP_CNTR_RX_SPINEL_TOTAL, &NcpBase::GetPropertyHandler_NCP_CNTR },
     { SPINEL_PROP_CNTR_RX_SPINEL_ERR, &NcpBase::GetPropertyHandler_NCP_CNTR },
 
+    { SPINEL_PROP_CNTR_TX_PKT_UNICAST, &NcpBase::GetPropertyHandler_NCP_CNTR },
```

Comment:
I suggest the new `TX_PKT` ones to be added after the other `TX_PKT` counters and the new `RX_PKT` ones after the other `RX_PKT` counters.